packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal-content-items/jurisdiction-modal-card/verification-status-banner.tsx

```diff
@@ -0,0 +1,200 @@
+
+            return null;
+        }
+        if (poi_or_poa_not_submitted) {
+            // if poi or poa is not submitted
+            return (
+                <div className={`${card_classname}__footer--none`}>
+                    <Text as='p' size='xxs' align='center' color={'prominent'}>
+                        <Localize i18n_default_text='Proof of identity and address are required' />
+                    </Text>
+                </div>
+            );
+        } else if (type_of_card === 'vanuatu') {
+            if (poi_poa_verified_for_vanuatu) {
+                return null; //both verified-no banner
+            } else if (poi_acknowledged_for_vanuatu && poa_acknowledged && !poi_poa_verified_for_vanuatu) {
+                //one is pending and other is verified
+                return (
+                    <div className={`${card_classname}__verification-status`}>
+                        <div className={`${card_classname}__verification-status--pending`}>
+                            <Text size='xxxs' color={'prominent'}>
+                                <Localize i18n_default_text='Pending verification' />
+                            </Text>
+                        </div>
+                    </div>
+                );
+            } else if (poi_acknowledged_for_vanuatu && need_poa_resubmission) {
+                // poa is rejected,suspected, failed-resubmit
+                return (
+                    <div className={`${card_classname}__verification-status`}>
+                        <div className={`${card_classname}__verification-status--POA_POI`}>
+                            <Text size='xxxs' color={'white'}>
+                                <Localize i18n_default_text='Check your proof of address' />
+                            </Text>
+                        </div>
+                    </div>
+                );
+            } else if (need_poi_for_vanuatu && poa_acknowledged) {
+                return (
+                    <div className={`${card_classname}__footer--none`}>
+                        <Text as='p' size='xxs' align='center' color={'prominent'}>
+                            <Localize i18n_default_text='Proof of identity required' />
+                        </Text>
+                    </div>
+                );
+            } else if (need_poi_for_vanuatu && need_poa_submission) {
+                return (
+                    <div className={`${card_classname}__verification-status`}>
+                        <div className={`${card_classname}__verification-status--POA_POI`}>
+                            <Text size='xxxs' color={'white'}>
+                                <Localize i18n_default_text='Check your proof of identity and address' />
+                            </Text>
+                        </div>
+                    </div>
+                );
+            }
+        } else if (type_of_card === 'bvi' || type_of_card === 'labuan' || type_of_card === 'maltainvest') {
```

Comment:
For multiple or checks, you can use the `Array.includes` way of writing to make it less repetitive and easier to read
```suggestion
        } else if (['bvi', 'labuan', 'maltainvest'].includes(type_card)) {
```